snapshots/output/basic-project/lib/more.dart

```diff
@@ -35,6 +35,7 @@
   class Animal with SleepMixin {
 //      ^^^^^^ definition scip-dart pub dart_test 1.0.0 lib/more.dart/Animal#
 //      documentation ```dart
+//      relationship scip-dart pub dart_test 1.0.0 lib/more.dart/SleepMixin# implementation
```

Comment:
we're running relationship indexing on `basic-project` now, snapshots were not previously generated due to failing CI that neglected to run on previous PRs